spec/lib/dispatch-rider/airbrake_error_handler_spec.rb

```diff
@@ -3,11 +3,11 @@
 describe DispatchRider::AirbrakeErrorHandler do
 
   describe ".call" do
-    let(:message){ DispatchRider::Message.new(subject: "TestMessage", body: "foo" )}
-    let(:exception){ Exception.new("Something went terribly wrong") }
+    let(:message) { DispatchRider::Message.new(subject: "TestMessage", body: "foo") }
+    let(:exception) { Exception.new("Something went terribly wrong") }
 
     example do
-      Airbrake.should_receive(:notify).with(exception, controller: "DispatchRider", action: "TestMessage", parameters: { subject: "TestMessage", body: "foo" }, cgi_data: anything)
+      expect(Airbrake).to receive(:notify).with(exception, controller: "DispatchRider", action: "TestMessage", parameters: {subject: "TestMessage", body: "foo"}, cgi_data: anything)
```

Comment:
Line is too long. [181/120]<br>Space inside { missing.<br>Space inside } missing.